ParseCNVQC/parse_cnv_qc.py

```diff
@@ -62,6 +62,51 @@ def make_mail(today, daysago, attachment, run_status):
     send_email(settings.email_from, settings.email_to, subject, text, attachment)
 
 
+def include_sample_number(folder, rawfolder, projects, warnings):
```

Comment:
rename naar b.v. get_num_samples_per_run_from_samplesheet of iets in die trand, wat direct impliceert wat de functie doet.